bqplot/marks.py

```diff
@@ -423,11 +427,10 @@ class Scatter(Mark):
         Labels for the points of the chart
     display_names: bool (default: True)
         Controls whether names are displayed for points in the scatter
-    fill, drag_color, names_unique, enable_move, enable_add,
-    enable_delete, restrict_x, restrict_y, update_on_move.
 
     Data Attributes
```

Comment:
`**Data Attributes**` ?  There are also a lot of other non-supported headings below.